### ISSUE TITLE
This seems wrong. . .

### ISSUE THREAD

**Issue by gmyenni** [weecology/PortalData#334]
```
     year month day mintemp maxtemp meantemp precipitation locally_measured battery_low
1051 1982    11  16     8.5      16       NA             0             TRUE          NA
1052 1982    11  17     6.5     113       NA             0             TRUE          NA
1053 1982    11  18     4.0      14       NA             0             TRUE          NA
```

**Comment by gmyenni** [comment]
113 is also the value in the raw data. I feel like it's probably 13, but even better if we can see the graph.

**Comment by skmorgane** [comment]
I checked the archive. I only saw sheets for 1987. I agree with your assessment though. In november between two other days where max temp was < 20? Definitely a typo. The value is probably either 11 or 13 (more likely 13 with a stuck 1 key). I'd say check the other stations in the area just incase they also show something anomalous to dot is and cross ts and then maybe correct and flag?

**Comment by gmyenni** [comment]
Closed via  #336
